packages/shared/client/ReplayClient.ts

```diff
@@ -877,75 +883,51 @@ export class ReplayClient implements ReplayClientInterface {
     }
   }
 
-  async _waitForPointToBeLoaded(point: ExecutionPoint): Promise<void> {
+  async waitForPointToBeInFocusRange(point: ExecutionPoint): Promise<void> {
```

Comment:
Not specific to this PR, and you all may well agree since I'm sure changing this now would be hard, but the fact that this hangs indefinitely if the point isn't in range really seems dangerous to me from an API-design standpoint.

Reply:
During the chat Holger and I had yesterday, the idea of adding a timeout to this came up. Guess that didn't make it into this PR but seems like it might be worth adding as a follow up.